src/Module/Admin/Logs/View.php

```diff
@@ -81,13 +80,35 @@ public static function content(array $parameters = [])
 			}
 		}
 		return Renderer::replaceMacros($t, [
-			'$title'         => DI::l10n()->t('Administration'),
-			'$page'          => DI::l10n()->t('View Logs'),
+			'$title' => DI::l10n()->t('Administration'),
+			'$page'  => DI::l10n()->t('View Logs'),
+			'$l10n'  => [
+				'Search'                => DI::l10n()->t('Search'),
+				'Search_in_logs'        => DI::l10n()->t('Search in logs'),
+				'Show_all'              => DI::l10n()->t('Show all'),
+				'Date'                  => DI::l10n()->t('Date'),
+				'Level'                 => DI::l10n()->t('Level'),
+				'Context'               => DI::l10n()->t('Context'),
+				'Message'               => DI::l10n()->t('Message'),
+				'ALL'                   => DI::l10n()->t('ALL'),
+				'View_details'          => DI::l10n()->t('View details'),
+				'Click_to_view_details' => DI::l10n()->t('Click to view details'),
+				'Event_details'         => DI::l10n()->t('Event details'),
+				'Data'                  => DI::l10n()->t('Data'),
+				'Source'                => DI::l10n()->t('Source'),
+				'File'                  => DI::l10n()->t('File'),
+				'Line'                  => DI::l10n()->t('Line'),
+				'Function'              => DI::l10n()->t('Function'),
+				'UID'                   => DI::l10n()->t('UID'),
+				'Process_ID'            => DI::l10n()->t('Process ID'),
+				'Close'                 => DI::l10n()->t('Close'),
+			],
 			'$data'          => $data,
 			'$q'             => $search,
 			'$filters'       => $filters,
 			'$filtersvalues' => $filters_valid_values,
 			'$error'         => $error,
+			'$baseurl'       => DI::baseUrl()->get(true),
```

Comment:
What is the baseurl needed for?

Reply:
mmh.. nothing I think.. I copy-pasted some code from View and forgot to remove that.. let me see..

Reply:
It can be needed to use absolute URLs in templates, and as such it is added by default in all templates, so this instance is superfluous on multiple counts.